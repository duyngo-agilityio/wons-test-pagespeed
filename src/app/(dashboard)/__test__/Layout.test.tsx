import { render } from '@testing-library/react';
import RootLayout from '../layout';

jest.mock('@/layouts', () => ({
  Sidebar: () => <div>Sidebar</div>,
}));

describe('RootLayout', () => {
  it('should render correctly and match snapshot', async () => {
    const { container } = render(
      <RootLayout>
        <div>Content</div>
      </RootLayout>,
    );

    expect(container).toMatchSnapshot();
  });
});
