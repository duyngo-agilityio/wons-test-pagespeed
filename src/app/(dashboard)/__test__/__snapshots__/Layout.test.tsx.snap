// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootLayout should render correctly and match snapshot 1`] = `
<div>
  <div
    class="lg:flex"
  >
    <div>
      Sidebar
    </div>
    <div
      class="flex-1 pt-7.5 base:px-6 md:pl-7.5 md:pr-7 mb-8 max-w-[1324px] mx-auto"
    >
      <div>
        Content
      </div>
    </div>
  </div>
</div>
`;
